conf/app.routes

```diff
@@ -34,4 +34,4 @@ GET         /trader/:ern/draft/:draftId/confirmation         controllers.Confirm
 ->          /                                            exportInformation.Routes
 ->          /                                            destination.Routes
 ->          /                                            guarantor.Routes
-->          /                                            exportInformation.Routes
+->          /                                            importInformation.Routes
```

Comment:
This probably wasn't intentional

Reply:
It should be importInformation, exportInformation is already on line 34